### PR TITLE
Homogenize Example

### LINKED ISSUE #11: Create homogenous micro rollup examples
The micro rollup examples used as template via CLI right now are confusing and quite different to each other and for a new user can be very confusing at first sight to understand the proper way to build MRUs.

Specifically following created the confusion
- [ ] State creation of the MRU & state wrapping
- [ ] Interaction with the MRU ,specifically the way to interact from frontend is not provided
- [ ] MRU file structure

Suggestion would be to use a simple and single structure of the different components of the MRU and then give unique examples of state and ways of interaction.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Closes #11 and standardizes the example template.

```sh
.
├── .env.example
├── .gitignore
├── Dockerfile
├── genesis-state.json
├── package-lock.json
├── package.json
├── src
│   ├── index.ts ## -> starting point, everything gets imported here.
│   ├── server.ts ## -> server setup if any in the example.
│   ├── cli.ts ## -> CLI interaction setup if any in the example.
│   ├── contract ## -> place to keep your utility Contracts like Hook and Bridge
│   │   └──Contract.sol 
│   ├── stackr
│       ├── machine.ts ## -> preferred place to keep your State Machine(s) and export from
│       ├── mru.ts ## -> place to initialize your MicroRollup
│       ├── schemas.ts ## -> one place to create and export all schemas from
│       ├── state.ts ## -> file to define your State class, can be omitted if state is trivial.
│       └── transitions.ts ## -> one place to store all your transitions & hooks _(hooks can have seaprate hooks.ts file too.)_
├── stackr.config.ts
├── tests
│   └── some.test.ts
└── tsconfig.json
```

### CODE REVIEW

File: bridge/src/stackr/schemas.ts
Comment (Dhruv-2003) [inline bridge/src/stackr/schemas.ts:1]: The action schema identifier is different from transition name , might create confusion

File: empty/src/stackr/machine.ts
Comment (Dhruv-2003) [inline empty/src/stackr/machine.ts:1]: do we need a separate state.ts & transitions.ts file here as well just with comments as we have in all of the others

**Review by Dhruv-2003 — COMMENTED** [review]
The new structure looks good and is similar across all of the files except empty. 

One issue I still have is the action identifier and the transition name , while I understand they don't need to be same, I still think across the examples , there are different versions of this thing and some of the examples don't have stfSchemaMap.

Interaction is sorted as well. On a side note if we can have something like scaffold-mru , it would be the best example for most of the developers.

**Comment by aashutoshrathi** [comment]
> On a side note if we can have something like scaffold-mru , it would be the best example for most of the developers.
I'm building that.

File: empty/src/stackr/machine.ts
Comment (aashutoshrathi) [reply empty/src/stackr/machine.ts:1]: I wanted to opinionate it as little as possible since this example was meant to set the user free.

Comment (Dhruv-2003) [reply empty/src/stackr/machine.ts:1]: Yeah , in that sense it's fine , we have given the basic building block

File: bridge/src/stackr/schemas.ts
Comment (aashutoshrathi) [reply bridge/src/stackr/schemas.ts:1]: I get your point, but having them as different might be beneficial as users assume they can pass actionSchema Identifier, in submitAction. Still, they might not be the same as "transition name".
For cases like [one schema being used in multiple transitions](https://github.com/stackrlabs/unification-cup/blob/68a9a73ba435a83097714fe7ea9a04d0a46a52d9/src/index.ts#L30), promoting this practice of keeping the same Action Schema Identifier and transition might not be good.

Comment (Dhruv-2003) [reply bridge/src/stackr/schemas.ts:1]: Agreed

File: token-transfer/src/server.ts
Comment (Dhruv-2003) [inline token-transfer/src/server.ts:1]: Just remembered , are we adding the info endpoint here or in the scaffold example ?

**Review by Dhruv-2003 — APPROVED** [review]

File: token-transfer/src/server.ts
Comment (aashutoshrathi) [reply token-transfer/src/server.ts:1]: I think in the scaffold

**Pull request merged**